src/forms/components/inputs/_input.scss

```diff
@@ -146,8 +146,8 @@
   color: $color-gray-70;
 }
 
-.errorLegend legend, .errorLabel label {
-  color: #da1e28
+.errorLabel > div label {
```

Comment:
Is this change related to the EncounterList enhancements?

Reply:
The covid feature branch is behind dev, a rebase will fix this